### ISSUE TITLE
Kitematic - Issue while creating Virtualbox Network Adapters which leads to multiple deltion and installtion of docker machines

### ISSUE THREAD

**Issue by ynagarjuna1995** [kitematic/kitematic#1150]
![image](https://cloud.githubusercontent.com/assets/5663901/10795322/2f354ece-7dbf-11e5-81d9-dfc3e8211d57.png)

Kitematic is creating multiple Virtual box network interfaces and a pop-up is thrown on windows 10 everytime which sometimes leads to problems like 

- Kitematic Hanging at 99%
- Ceritficate errors 

And once If I stop my `default` machine again when I attempt to start or open kitematic I am getting same pop-up and ending up with a weird error like the 2 erros as stated above.

**Comment by FrenchBen** [comment]
@ynagarjuna1995 Can you run the following to see if the `default` VM was created properly:
`docker-machine ls`

You can also see the list of VMs created by opening VirtualBox

**Comment by ynagarjuna1995** [comment]
@FrenchBen 
Yeah It gives a message after 5 minutes or 60 Retries which might be default retries set up by the Kitematic team after that it throws a message which I encountered Sorry I couldn't able provide that message right now 

To solve the problem I have uninstalled Virtualbox network adapters,Docker Tooolbox and all the files and certificates related to docker and Virtualbox and again installed 

After that I was able to work with docker properly..

##Feature : 

Is it possible to boot the different machine with kitematic like `dev` or another docker machine Right now kitematic creates `default` machine and bootups that machine by default I want to boot `dev` machine by default or else an option where in which kitematic asks user which docker-machine to boot.

**Comment by FrenchBen** [comment]
@ynagarjuna1995 that is on our list of requests, which is allowing the user to select the VM that Kitematic will be using when it boots up

**Comment by FrenchBen** [comment]
As we've released a new Docker Toolbox and VirtualBox was updated, could you re-install and check if VirtualBox is still trying to add the multiple net interfaces

**Comment by ynagarjuna1995** [comment]
![image](https://cloud.githubusercontent.com/assets/5663901/10980604/f4e2eb8a-8428-11e5-9dd8-2bc13d46838d.png)
@FrenchBen  Improvement over past release but still creating a extra network adapter `Ethernet 2` instead of using  `Virtual Box Host-Only Network` which is provided by default by Oracle 

you can see the pop-up which says 

      cannot rename this connection a connection with specified 
      name already exists specify a different name 

Upon  Pressing  `OK` I guess or before (Sorry unable to recollect this )
It has `Ethernet 2` network adapter.

This will cause serious problems release to release because 
We have to uninstall previous Docker Toolbox for installing new release when I install Toolbox it is uninstalling only Toolbox excluding Virtualbox

When a noob tries to install a Toolbox without unchecking virtualbox it may create problems and Docker Toolbox might create multiple adpaters again and again ..

**Comment by FrenchBen** [comment]
Unfortunately this seems to be more of an issue with VirtualBox not properly setting up the Host-Only network rather than Toolbox

If you look at this bug list, you can see that this is an on-going issue:
https://www.virtualbox.org/ticket/14040

Until a proper patch is implemented by Virtualbox there isn't much we'll be able to do.
